ravenframework/Metrics/metrics/DSS.py

```diff
@@ -23,7 +23,7 @@
 
 #Internal Modules------------------------------------------------------------------------------------
 from .MetricInterface import MetricInterface
-from utils import InputData, InputTypes
+from ...utils import InputData, InputTypes
```

Comment:
@joshua-cogliati-inl which is a better way to import the module, "ravenframework.utils" or "...utils"? I think I prefer to using "ravenframework.utils"

Reply:
Hm, well, one reason I used ...utils was so I didn't have to change it when framework got renamed.  Right now ravenframework.whatever is a good warning that that section of code has something strange happening to the python path (like in CodeInterfaces or in CustomModes). For this pull request I recommend keeping the relative paths, but I have no strong opinions on this for future additions.